api/document/management/commands/import_xml_doc.py

```diff
@@ -17,6 +18,14 @@ def fetch_policy(identifier: str):
         return Policy.objects.filter(omb_policy_id=identifier).first()
 
 
+def import_xml_doc(policy, xmlstream):
+    parsed_data = AkomaNtosoParser().parse(xmlstream)
+    cursor = DocCursor.new_tree('', policy=policy)
+    serializer = DocCursorSerializer(cursor, data=parsed_data)
+    serializer.is_valid(raise_exception=True)
```

Comment:
I'm betting this raises some gnarly, but data-rich exceptions. Not necessary for this round, but we should think about ways to surface them to users.